hack/externalTools.go

```diff
@@ -0,0 +1,7 @@
+package hack
+
+import (
+	"k8s.io/code-generator/pkg/util"
+)
+
+var _ = util.CurrentPackage
```

Comment:
@ryantxu Are you ok with this workaround to ensure `go mod tidy` doesn't try to delete `k8s.io/code-generator` from go.mod?